docker/run_both_servers.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]; then
+   echo "Usage: $0 <transform=true/false>"
+   exit 1
+fi
+
+cd /root
+
+java --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED -Dlogback.configurationFile=file:logging.xml -cp "flight-server-all.jar" org.m4d.adp.flight_server.ExampleFlightServer relay $1 0.0.0.0 12232 localhost 12233 &
+java --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED -Dlogback.configurationFile=file:logging.xml -cp "flight-server-all.jar" org.m4d.adp.flight_server.ExampleFlightServer example localhost 12233
```

Comment:
I think the arguments should be modified according to the changes of argument parsing in `ExampleFlightServer` (as in `server.sh`).

Reply:
Thanks. Will fix